akka-actor-typed/src/main/scala/akka/actor/typed/Behavior.scala

```diff
@@ -54,6 +53,10 @@ sealed abstract class Behavior[T] { behavior ⇒
   final def orElse(that: Behavior[T]): Behavior[T] = new OrElseBehavior[T](this, that)
 }
 
+trait BehaviorTag {
```

Comment:
Would be better to put this in an abstract class if possible since it will be an `invokevirtual` rather than an `invokeinterface`. Then I'd make all implementations of `tag` `final` to make sure that things can be properly optimized.

Reply:
@viktorklang I think @johanandren have implemented this in https://github.com/akka/akka/pull/26541 :)

Reply:
Yeah, thanks for the idea @hepin1989 gave a nice throughput improvement!